doc/sphinx-guides/source/developers/coding-style.rst

```diff
@@ -2,26 +2,38 @@
 Coding Style
 ============
 
-Like all development teams, the `Dataverse developers at IQSS <https://dataverse.org/about>`_ have their habits and styles when it comes to writing code. Let's attempt to get on the same page. :)
-
-.. contents:: |toctitle|
-	:local:
```

Comment:
We want a table of contents on every non-index page. The rule is at http://guides.dataverse.org/en/4.7/developers/documentation.html#table-of-contents and was added as part of #3796.